src/include/network/postgres/statement.h

```diff
@@ -148,6 +156,7 @@ class Statement {
   // same query text. The exception to this that DDL changes can break these cached objects.
   std::unique_ptr<optimizer::OptimizeResult> optimize_result_ = nullptr;              // generated in the Bind phase
   std::unique_ptr<execution::compiler::ExecutableQuery> executable_query_ = nullptr;  // generated in the Execute phase
+  uint64_t executable_query_timestamp_;                                               // generated with executable_query
```

Comment:
Actually, if the executable query timestamp is a property of the executable query, should it simply be stored in the `ExecutableQuery` class? Then you also don't need a default timestamp, if you have a valid `ExecutableQuery` then you have a valid timestamp.